Extension/CHANGELOG.md

```diff
@@ -1,6 +1,22 @@
 # C/C++ for Visual Studio Code Change Log
 
-## Version 0.17.2: May 21, 2018
+## Version 0.17.5-insiders: June 7, 2018
+* Change inactive regions from gray to translucent. [#1907](https://github.com/Microsoft/vscode-cpptools/issues/1907)
```

Comment:
Any edits/additions?

Reply:
There was some IntelliSense creation changed, but I wasn't sure how to describe it. And the config provider change. And the additional config args caching.